SCons/SConf.py

```diff
@@ -714,6 +714,12 @@ def TryRun(self, text, extension ):
         if ok:
             prog = self.lastTarget
             pname = prog.get_internal_path()
+            if sys.platform == "win32" and os.sep == "/":
```

Comment:
Would it make sense to add a SCons.Util function to determine you're using msys python?